@copilot+claude-sonnet-4.6 Could you fix the following error in CI?

> -- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
ERROR gdsfactory/samples/sample_routing_auto_taper.py - ImportError while importing test module '/Users/runner/work/gdsfactory/gdsfactory/gdsfactory/samples/sample_routing_auto_taper.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/homebrew/Cellar/python@3.13/3.13.13/Frameworks/Python.framework/Versions/3.13/lib/python3.13/importlib/__init__.py:88: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
gdsfactory/samples/sample_routing_auto_taper.py:3: in <module>
    from gpdk import PDK
E   ModuleNotFoundError: No module named 'gpdk'
====== 1555 passed, 2 skipped, 1 xfailed, 115 warnings, 1 error in 37.43s ======
make: *** [test] Error 1
Error: Process completed with exit code 2.

Remember to also run the pre-commit hooks afterwards and fix any errors.

--- a/gdsfactory/samples/sample_routing_auto_taper.py
+++ b/gdsfactory/samples/sample_routing_auto_taper.py
@@ -1,7 +1,5 @@
 """Slot sample."""
 
-from gpdk import PDK
-
 import gdsfactory as gf
 
 
@@ -44,6 +42,8 @@ def sample_routing_auto_taper_trenches() -> gf.Component:
 
 
 if __name__ == "__main__":
+    from gpdk import PDK
+
     PDK.activate()
 
     c = sample_routing_auto_taper_trenches()

